llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -1416,14 +1560,46 @@ bool RISCVFrameLowering::restoreCalleeSavedRegisters(
   // first in the epilogue. It increases the opportunity to avoid the
   // load-to-use data hazard between loading RA and return by RA.
   // loadRegFromStackSlot can insert multiple instructions.
+  //
+  //
+  // We first change the restore order for scalar and vector
+  // callee-saved registers as the layout shown below:
+  //
+  // Epilog restore order (original):
+  //     ----------------------------
+  //      RVV objects
```

Comment:
The epilog doesn't "restore" RVV objects. Other than adjusting the stack pointer. I'm not sure what this is illustrating